lisp/plugins/gst_backend/elements/wayland_sink.py

```diff
@@ -0,0 +1,43 @@
+# This file is part of Linux Show Player
+#
+# Copyright 2024 Francesco Ceruti <ceppofrancy@gmail.com>
+#
+# Linux Show Player is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Linux Show Player is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Linux Show Player.  If not, see <http://www.gnu.org/licenses/>.
+
+from PyQt5.QtCore import QT_TRANSLATE_NOOP
+
+from lisp.backend.media_element import ElementType, MediaType
+from lisp.plugins.gst_backend.gi_repository import Gst
+from lisp.plugins.gst_backend.gst_element import GstMediaElement
+from lisp.plugins.gst_backend.gst_properties import GstProperty
+
+
+class WaylandSink(GstMediaElement):
+    ElementType = ElementType.Output
```

Comment:
1. if a user opens a newly created Media (Video) Cue, then opens the "Edit Pipeline" dialog, it is possible to specify the "Wayland Video" element as the Output (instead of "System Out", "ALSA Out", ...), leading to (once the dialog closed) "Wayland Video" element appearing twice in the pipeline.

2. if a user removes the "Wayland Video" element from the pipeline of a Media (Video) Cue, then closes the "Edit Pipeline" dialog, upon reopening it, it is no longer possible to add the "Wayland Video" element back.

Both can be resolved by setting this to `ElementType.Plugin`.

Admittedly this is perhaps *conceptually* wrong, but it works!